internal/dump/installedobjects_test.go

```diff
@@ -15,30 +16,87 @@ import (
 
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
 
+	"github.com/elastic/elastic-package/internal/elasticsearch"
 	estest "github.com/elastic/elastic-package/internal/elasticsearch/test"
 	"github.com/elastic/elastic-package/internal/files"
 )
 
-func TestInstalledObjectsDumpAll(t *testing.T) {
-	client := estest.ElasticsearchClient(t, "./testdata/elasticsearch-mock-dump-apache")
-	outputDir := t.TempDir()
-	dumper := NewInstalledObjectsDumper(client, "apache")
+func TestDumpInstalledObjects(t *testing.T) {
+	// Files for each suite are recorded automatically on first test run.
+	// To add a new suite:
+	// - Configure it here.
+	// - Install the package in a running stack.
+	// - Configure environment variables for this stack (eval "$(elastic-package stack shellinit)").
```

Comment:
I might have missed something introduced in the previous PR, but does it require the Elasticsearch to be up or does it still operate on mocks?